### DISCUSSION TITLE
Create a universal linux Flatpak container for Firefly  (the most widely used container on linux)

### DISCUSSION THREAD

**Discussion by jbingram** [iotaledger/firefly#1572] [Features / Enhancements]
Thx for your awesome job, much appreciated.

## Description (Briefly describe the feature that you are requesting.)

Create linux Flatpak sandboxing for Trinity and inform the general flathub.org repository. https://flathub.org/

## Motivation (Explain why this feature is needed.)

- Flatpak apps are sandboxed apps that are more secured than traditional linux packaging systems (e.g. .deb, .rpm, .tar.gz). The users can run Trinity software in isolation from the rest of the linux system. 
- Flatpak is universal, as it allows a single application build to be installed and run on virtually any Linux distribution. 
- Flatpak apps can also be used in combination with Flathub, a centralized service for distributing applications on all distributions. This makes it possible for application developers to target the entire Linux desktop market from one place. 
- Flatpak apps are natively integrated into all Linux desktops, tablets and now smartphones, so that users can easily browse, install, run and use Flatpak applications through their existing desktop environment and tools. 
-  Forward-compatibility: the same Flatpak can be run on different versions of the same distribution, including versions that haven’t been released yet. This doesn’t require any changes or management by application developers.
- Maintained platforms: called runtimes, these contain collections of dependencies, which can be used by applications, and which can take a lot of the work out of application development.
- Bundling: this allows application developers to ship almost any dependency or library as part of their application. This gives complete control over which software is used to build applications.
- Consistent application environments: because these are the same across devices, applications perform as intended. This also makes it easier to identify bugs and to do testing.
- Finally, while Flatpak does provide a centralized service for distributing applications, it also allows decentralized hosting and distribution, so that application developers or downstreams can host their own applications and application repositories.

## Requirements

Quite easy. See documentation at https://docs.flatpak.org/en/latest/

## Are you planning to do it yourself in a pull request?

No. as it is important for trust that the official flatpak of Trinity on Flathub.org  comes from an official repo and from official IF members

Thx for considering my request and keep up the good work

**Comment by iuvbio** [comment]
I would also very much appreciate a flatpak. Not everyone uses snap and as far as I see that's the only way to install firefly on linux currently?

**EDIT**: just saw that there is an AppImage as well, I'm fine with that. Just need ledger support now..

**Comment by maxwellmattryan** [comment]
Hey @jbingram, I've closed this issue and put it into a discussion [here](https://github.com/iotaledger/firefly/discussions/1506). Thank you for the Flatpak request!

**Reply by jbingram** [reply]
Hi @maxwellmattryan , the link is dead. Also any news for this request? I'm on fedora silverblue on my primary computer and still have no access to my iota wallet. I guess the entire linux world interested in IOTA and who is inevitably migrating more and more to flatpak is confronted with the same problem.
